helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -100,3 +112,33 @@ internal class HelperMain : CliktCommand(name = ORTH_NAME, epilog = "* denotes r
         printStackTrace = stacktrace
     }
 }
+
+/**
+ * Read [ortFile] into an [OrtResult] and return it. Make sure that information about project scopes is available
+ * (by calling [OrtResult.withResolvedScopes]), so that it can be processed.
+ */
+fun CliktCommand.readOrtResult(ortFile: File): OrtResult {
+    log.debug { "Input ORT result file has SHA-1 hash ${HashAlgorithm.SHA1.calculate(ortFile)}." }
```

Comment:
Just as a not, I'm not sure how useful the SHA1 calculation is for the `helper-cli` module. It was mostly added for debugging "hand-over" issues in pipelines running ORT. So you might want to remove it here and below.

Reply:
I wasn't sure what the use case was so I have kept it, but I have removed it now.